### ISSUE TITLE
The testsuite doesn't run in parallel

### ISSUE THREAD

**Issue by vicuna** [ocaml/ocaml#7072]
**Original bug ID:** 7072
**Reporter:** @Drup
**Assigned to:** @shindere
**Status:** resolved (set by @xavierleroy on 2017-09-24T15:23:31Z)
**Resolution:** fixed
**Priority:** low
**Severity:** minor
**Target version:** later
**Fixed in version:** 4.06.0 +dev/beta1/beta2/rc1
**Category:** configure and build/install
**Monitored by:** @Drup @diml @hcarty

## Bug description

"make -j5 test" tends to crash because dependencies of tests are not specified.

While not critical, this would be nice to have, since running the testsuite seems like a highly parallel task.

**Comment by vicuna** [comment]
**Comment author:** @xavierleroy

In 4.06 and up, "make parallel" in the testsuite/ directory achieves good speedups with the help of GNU parallel.
